test/e2e/seeder/contracts/collectibles-factory.js

```diff
@@ -0,0 +1,356 @@
+const collectiblesFactory = {
```

Comment:
can we import this contract instead of copying it all across? I think it's accessible in the `test-dapp` package

Reply:
I've tried but I kept getting this error `window is not defined`.

Investigating further I think at the moment it is not possible, due to the way test-dapp package is built, as contract variables are not accessible from bundle. Here an explanation:
https://stackoverflow.com/questions/58434256/how-do-i-access-variables-functions-in-bundle-js

I think we could do this if I modify the test-dapp repo and made contract variables accessible from the outside. What do you think @PeterYinusa ? Should I try this option?

Reply:
I think you should be able to get the file containing the abi and bytecode in `smart-contracts.js` with the following:

```
const constants = require('../../../node_modules/@metamask/test-dapp/dist/constants.json');
```

Then we can remove entire contracts directory.

Reply:
thank you v much @PeterYinusa that work successfully!!